src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/TemplatePathsCache/templates.php

```diff
@@ -1,3 +1,5 @@
-<?php return array (
+<?php
+
+return array(
```

Comment:
These files are fixture files used as comparison for the generated ones, so this change is wrong (it makes tests fail)